pallets/author-slot-filter/src/lib.rs

```diff
@@ -94,10 +94,18 @@ pub mod pallet {
 			// A context identifier for grabbing the randomness. Consists of three parts
 			// - The constant string *b"filter" - to identify this pallet
 			// - The index `i` when we're selecting the ith eligible author
+			// we take the first 2 bytes of index.to_be_bytes
+			// we take the first 4 bytes of seed.to_be_bytes
 			// - The relay parent block number so that the eligible authors at the next height
 			//   change. Avoids liveness attacks from colluding minorities of active authors.
 			// Third one may not be necessary once we leverage the relay chain's randomness.
```

Comment:
These comments are outdated.

The relay block number is no longer hard coded. Instead some `seed` is passed into this function. The line you added is better.